DockerEnvironmentBackupManager  ERROR 'java.lang.InterruptedException: null'

Got these exceptions in logs:
```
2017-02-21 14:10:06,312[aceSharedPool-0]  [ERROR] [DockerEnvironmentBackupManager 315]  - null
java.lang.InterruptedException: null
        at java.lang.Object.wait(Native Method) ~[na:1.8.0_111-internal]
        at java.lang.UNIXProcess.waitFor(UNIXProcess.java:412) ~[na:1.8.0_111-internal]
        at org.eclipse.che.api.core.util.ProcessUtil.executeAndWait(ProcessUtil.java:117) ~[che-core-api-core-5.3.0.jar:5.3.0]
        at com.codenvy.machine.backup.DockerEnvironmentBackupManager.executeCommand(DockerEnvironmentBackupManager.java:506) [codenvy-machine-hosted-5.3.0.jar:5.3.0]
        at com.codenvy.machine.backup.DockerEnvironmentBackupManager.restoreBackupInsideLock(DockerEnvironmentBackupManager.java:308) [codenvy-machine-hosted-5.3.0.jar:5.3.0]
        at com.codenvy.machine.backup.DockerEnvironmentBackupManager.restoreWorkspaceBackup(DockerEnvironmentBackupManager.java:194) [codenvy-machine-hosted-5.3.0.jar:5.3.0]
        at com.codenvy.machine.RemoteDockerNode.bindWorkspace(RemoteDockerNode.java:94) [codenvy-machine-hosted-5.3.0.jar:5.3.0]
        at com.codenvy.machine.agent.launcher.MachineInnerRsyncAgentLauncherImpl.launch(MachineInnerRsyncAgentLauncherImpl.java:68) [codenvy-machine-hosted-5.3.0.jar:5.3.0]
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes.launchAgents(WorkspaceRuntimes.java:695) [che-core-api-workspace-5.3.0.jar:5.3.0]
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes$MachineAgentsLauncher.started(WorkspaceRuntimes.java:1048) [che-core-api-workspace-5.3.0.jar:5.3.0]
        at org.eclipse.che.api.environment.server.CheEnvironmentEngine.startEnvironmentQueue(CheEnvironmentEngine.java:816) [che-core-api-workspace-5.3.0.jar:5.3.0]
        at org.eclipse.che.api.environment.server.CheEnvironmentEngine.start(CheEnvironmentEngine.java:267) [che-core-api-workspace-5.3.0.jar:5.3.0]
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes.startEnvironmentAndPublishEvents(WorkspaceRuntimes.java:714) [che-core-api-workspace-5.3.0.jar:5.3.0]
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes.access$100(WorkspaceRuntimes.java:106) [che-core-api-workspace-5.3.0.jar:5.3.0]
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes$StartTask.call(WorkspaceRuntimes.java:979) [che-core-api-workspace-5.3.0.jar:5.3.0]
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes$StartTask.call(WorkspaceRuntimes.java:943) [che-core-api-workspace-5.3.0.jar:5.3.0]
        at org.eclipse.che.commons.lang.concurrent.CopyThreadLocalCallable.call(CopyThreadLocalCallable.java:30) [che-core-commons-lang-5.3.0.jar:5.3.0]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_111-internal]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_111-internal]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_111-internal]
        at java.lang.Thread.run(Thread.java:745) [na:1.8.0_111-internal]
```
Error message lacks information about what workspace and node caused error.
Probably backup/restore was timed out, if this is a case then please change loglevel to WARN and include workspaceID and node where workspace is running, don't print stacktrace.


**Reproduction Steps:**
Don't know

**Codenvy version:**   5.3.0

It doesn't fixed, I'm still getting such errors:
<details>

```
2017-03-27 12:49:04,769[ceSharedPool-10]  [INFO ] [o.e.c.a.w.s.WorkspaceManager 795]    - Workspace 'pl4net/DistributedSystems' with id 'workspace*******' is being stopped by user '****'
2017-03-27 12:49:04,771[aceSharedPool-8]  [ERROR] [DockerEnvironmentBackupManager 315]  - null
java.lang.InterruptedException: null
        at java.lang.Object.wait(Native Method) ~[na:1.8.0_111-internal]
        at java.lang.UNIXProcess.waitFor(UNIXProcess.java:412) ~[na:1.8.0_111-internal]
        at org.eclipse.che.api.core.util.ProcessUtil.executeAndWait(ProcessUtil.java:117) ~[che-core-api-core-5.5.0.jar:5.5.0]
        at com.codenvy.machine.backup.DockerEnvironmentBackupManager.executeCommand(DockerEnvironmentBackupManager.java:506) [codenvy-machine-hosted-5.5.0.jar:5.5.0]
        at com.codenvy.machine.backup.DockerEnvironmentBackupManager.restoreBackupInsideLock(DockerEnvironmentBackupManager.java:308) [codenvy-machine-hosted-5.5.0.jar:5.5.0]
        at com.codenvy.machine.backup.DockerEnvironmentBackupManager.restoreWorkspaceBackup(DockerEnvironmentBackupManager.java:194) [codenvy-machine-hosted-5.5.0.jar:5.5.0]
        at com.codenvy.machine.RemoteDockerNode.bindWorkspace(RemoteDockerNode.java:94) [codenvy-machine-hosted-5.5.0.jar:5.5.0]
        at com.codenvy.machine.agent.launcher.MachineInnerRsyncAgentLauncherImpl.launch(MachineInnerRsyncAgentLauncherImpl.java:68) [codenvy-machine-hosted-5.5.0.jar:5.5.0]
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes.launchAgents(WorkspaceRuntimes.java:695) [che-core-api-workspace-5.5.0.jar:5.5.0]
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes$MachineAgentsLauncher.started(WorkspaceRuntimes.java:1048) [che-core-api-workspace-5.5.0.jar:5.5.0]
        at org.eclipse.che.api.environment.server.CheEnvironmentEngine.startEnvironmentQueue(CheEnvironmentEngine.java:816) [che-core-api-workspace-5.5.0.jar:5.5.0]
        at org.eclipse.che.api.environment.server.CheEnvironmentEngine.start(CheEnvironmentEngine.java:267) [che-core-api-workspace-5.5.0.jar:5.5.0]
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes.startEnvironmentAndPublishEvents(WorkspaceRuntimes.java:714) [che-core-api-workspace-5.5.0.jar:5.5.0]
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes.access$100(WorkspaceRuntimes.java:106) [che-core-api-workspace-5.5.0.jar:5.5.0]
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes$StartTask.call(WorkspaceRuntimes.java:979) [che-core-api-workspace-5.5.0.jar:5.5.0]
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes$StartTask.call(WorkspaceRuntimes.java:943) [che-core-api-workspace-5.5.0.jar:5.5.0]
        at org.eclipse.che.commons.lang.concurrent.CopyThreadLocalCallable.call(CopyThreadLocalCallable.java:30) [che-core-commons-lang-5.5.0.jar:5.5.0]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [na:1.8.0_111-internal]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_111-internal]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_111-internal]
        at java.lang.Thread.run(Thread.java:745) [na:1.8.0_111-internal]
```
<details>

--- a/plugins/plugin-hosted/codenvy-machine-hosted/src/main/java/com/codenvy/machine/backup/DockerEnvironmentBackupManager.java
+++ b/plugins/plugin-hosted/codenvy-machine-hosted/src/main/java/com/codenvy/machine/backup/DockerEnvironmentBackupManager.java
@@ -312,14 +312,15 @@ public class DockerEnvironmentBackupManager implements EnvironmentBackupManager 
                     "Restoring of workspace " + workspaceId + " filesystem terminated due to timeout on "
                     + destAddress + " node.");
         } catch (InterruptedException e) {
-            LOG.error(e.getLocalizedMessage(), e);
+            Thread.currentThread().interrupt();
             throw new ServerException(
                     "Restoring of workspace " + workspaceId + " filesystem interrupted on " + destAddress + " node.");
         } catch (IOException e) {
-            LOG.error(e.getLocalizedMessage(), e);
-            throw new ServerException(
-                    "Restoring of workspace " + workspaceId + " filesystem terminated on " + destAddress + " node. "
-                    + e.getLocalizedMessage());
+            String error = "Restoring of workspace " + workspaceId +
+                           " filesystem terminated on " + destAddress + " node. "
+                           + e.getLocalizedMessage();
+            LOG.error(error, e);
+            throw new ServerException(error);
         } finally {
             lock.unlock();
             if (!restored) {

